src/pages/workspaces/workspace/notebooks/ExportNotebookModal.js

```diff
@@ -29,7 +35,6 @@ const ExportNotebookModal = ({ fromLauncher, onDismiss, printName, workspace })
   const signal = Utils.useCancellation()
   const { workspaces } = useWorkspaces()
 
-
```

Comment:
We generally leave 2 lines between named sections of a component, just to make it stand out.